src/components/manager/ManagerIngestionContent.js

```diff
@@ -2,15 +2,33 @@ import React from "react";
 import { useDispatch } from "react-redux";
 import { useNavigate } from "react-router-dom";
 
+import { ingestData, ingestReferenceMaterial, RESOURCE_EVERYTHING } from "bento-auth-js";
+
+import { useResourcePermissionsWrapper } from "@/hooks";
 import { submitIngestionWorkflowRun } from "@/modules/wes/actions";
 
+import ForbiddenContent from "./ForbiddenContent";
 import RunSetupWizard from "./RunSetupWizard";
 import RunSetupConfirmDisplay from "./RunSetupConfirmDisplay";
 
 const ManagerIngestionContent = () => {
     const dispatch = useDispatch();
     const navigate = useNavigate();
 
+    const { permissions, hasAttemptedPermissions } = useResourcePermissionsWrapper(RESOURCE_EVERYTHING);
+
+    // TODO: each workflow should have definitions for permissions scopes, so we can instead check if we can run at
+    //  least one workflow.
+
+    if (
+        hasAttemptedPermissions &&
+        !(permissions.includes(ingestData) || permissions.includes(ingestReferenceMaterial))
+    ) {
+        return (
+            <ForbiddenContent message="You do not have permission to view the ingestion wizard." />
+        );
+    }
```

Comment:
Suggestion: Here I feel a hook like `useHasResourcePermissions(resource: Resource, permissions: string[], checkAll=true)` would be useful. If `checkAll` is true, check that all the permissions are present, if it is false, check that at least one permission is present.

Reply:
I think I'll do that in a future PR as well, if that works